Manifest.cs

```diff
@@ -11,14 +11,14 @@
 
 [assembly: Feature(
     Id = Base,
-    Name = "Lombiq UIKit",
+    Name = "Lombiq UI Kit",
     Category = "Development",
     Description = "Module for reusable shapes containing text editors, dropdown editors, and potentially more complex editors."
 )]
 
 [assembly: Feature(
     Id = ShowCase,
-    Name = "Lombiq UIKit - Showcase page",
+    Name = "Lombiq UI Kit - Showcase page",
     Category = "Development",
-    Description = "You can reach the showcase page at /UIKitShowCase"
+    Description = "You can reach the showcase page under /UIKitShowCase"
```

Comment:
Mivel a showcase egy szó igazából, `/UIKitShowcase` kellene (meg ilyen névvel a controllerben is minden persze). De amúgy is inkonzisztens, mert szerepel "Showcase" és "ShowCase" is.